beekeeper-api/src/main/java/com/expediagroup/beekeeper/api/BeekeeperController.java

```diff
@@ -15,9 +15,23 @@
  */
 package com.expediagroup.beekeeper.api;
 
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.data.jpa.domain.Specification;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import com.expediagroup.beekeeper.core.model.HousekeepingMetadata;
+
 @RestController
 public class BeekeeperController {
-  
+
+  private HousekeepingEntityService housekeepingEntityService;
+
+  @RequestMapping("/tables")
```

Comment:
I think the endpoint `/housekeeping/tables` would be nicer. But ask Jose or other people for their opinion